doozer/doozerlib/olm/bundle.py

```diff
@@ -556,15 +556,28 @@ def list_of_manifest_files_to_be_copied(self):
             ))
         return files
 
+    @property
+    def operator_index_mode(self):
+        mode = self.runtime.group_config.operator_index_mode or 'ga'  # default when missing
+        if mode in {'pre-release', 'ga', 'ga-plus'}:
+            # pre-release: label for pre-release operator index (unsupported)
+            # ga: label for only this release's operator index
+            # ga-plus: label for this release's operator index and future release indexes as well
+            return mode
+        self.runtime.logger.warning(f'{mode} is not a valid group_config.operator_index_mode')
```

Comment:
should we raise an error here? although it was come from group.yaml and unlikely to be invalid